Source/JavaScriptCore/API/JSRemoteInspector.cpp

```diff
@@ -68,9 +77,61 @@ void JSRemoteInspectorSetLogToSystemConsole(bool logToSystemConsole)
     JSGlobalObjectConsoleClient::setLogToSystemConsole(logToSystemConsole);
 }
 
+static bool mainProcessHasEntitlement(WTF::ASCIILiteral entitlement, std::optional<audit_token_t> parentProcessAuditToken)
```

Comment:
Should this function be inside a `#if PLATFORM(COCOA)`?  Or does `audit_token_t` exist elsewhere?

NIT: Is the `WTF::` necessary?

Reply:
Yes, this should be wrapped in `#if PLATFORM(COCOA)`, this is why it doesn't compile on WinCairo (and likely others) currently.